.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -39,6 +39,7 @@ jobs:
     env:
       DOCKER_IMAGE_TEAM: registry.camunda.cloud/team-optimize/optimize
       DOCKER_IMAGE_DOCKER_HUB: camunda/optimize
+      DOCKER_INTERNAL_IMAGE_DOCKER_HUB: camunda/optimize-8
```

Comment:
I think this would push the image to the docker-hub public repo, the plan instead was to use a Google Artifact Registry to store the image and avoid confusion.

repo: [europe-west1-docker.pkg.dev/team-infosec/](http://europe-west1-docker.pkg.dev/team-infosec/)
example implementation: https://github.com/camunda/cve-registry/blob/main/.github/workflows/template-build.yaml

Reply:
@OmranAbazid - tagging you here, in case you missed this

Reply:
@RomanJRW this is already addressed in my PR.